services/venice-controller/src/main/java/com/linkedin/venice/controller/util/UpdateStoreUtils.java

```diff
@@ -0,0 +1,1239 @@
+        readComputationEnabled,
+        updatedStore::setReadComputationEnabled);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        NATIVE_REPLICATION_ENABLED,
+        nativeReplicationEnabled,
+        updatedStore::setNativeReplicationEnabled);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        ACTIVE_ACTIVE_REPLICATION_ENABLED,
+        activeActiveReplicationEnabled,
+        updatedStore::setActiveActiveReplicationEnabled);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        PUSH_STREAM_SOURCE_ADDRESS,
+        pushStreamSourceAddress,
+        updatedStore::setPushStreamSourceAddress);
+    addToUpdatedConfigs(updatedConfigs, BACKUP_STRATEGY, backupStrategy, updatedStore::setBackupStrategy);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        AUTO_SCHEMA_REGISTER_FOR_PUSHJOB_ENABLED,
+        autoSchemaRegisterPushJobEnabled,
+        updatedStore::setSchemaAutoRegisterFromPushJobEnabled);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        HYBRID_STORE_DISK_QUOTA_ENABLED,
+        hybridStoreDiskQuotaEnabled,
+        updatedStore::setHybridStoreDiskQuotaEnabled);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        BACKUP_VERSION_RETENTION_MS,
+        backupVersionRetentionMs,
+        updatedStore::setBackupVersionRetentionMs);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        NATIVE_REPLICATION_SOURCE_FABRIC,
+        nativeReplicationSourceFabric,
+        updatedStore::setNativeReplicationSourceFabric);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        LATEST_SUPERSET_SCHEMA_ID,
+        latestSupersetSchemaId,
+        updatedStore::setLatestSuperSetValueSchemaId);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        MIN_COMPACTION_LAG_SECONDS,
+        minCompactionLagSeconds,
+        updatedStore::setMinCompactionLagSeconds);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        MAX_COMPACTION_LAG_SECONDS,
+        maxCompactionLagSeconds,
+        updatedStore::setMaxCompactionLagSeconds);
+    addToUpdatedConfigs(updatedConfigs, MAX_RECORD_SIZE_BYTES, maxRecordSizeBytes, updatedStore::setMaxRecordSizeBytes);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        MAX_NEARLINE_RECORD_SIZE_BYTES,
+        maxNearlineRecordSizeBytes,
+        updatedStore::setMaxNearlineRecordSizeBytes);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        UNUSED_SCHEMA_DELETION_ENABLED,
+        unusedSchemaDeletionEnabled,
+        updatedStore::setUnusedSchemaDeletionEnabled);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        BLOB_TRANSFER_ENABLED,
+        blobTransferEnabled,
+        updatedStore::setBlobTransferEnabled);
+    addToUpdatedConfigs(
+        updatedConfigs,
+        STORAGE_NODE_READ_QUOTA_ENABLED,
+        storageNodeReadQuotaEnabled,
+        updatedStore::setStorageNodeReadQuotaEnabled);
+    addToUpdatedConfigs(updatedConfigs, REGULAR_VERSION_ETL_ENABLED, regularVersionETLEnabled, regularVersionETL -> {
```

Comment:
There is some potential of adding blank likes b/w logical groups, but I don't think it is needed here since almost all configs are logically different. It is also easier to modify when adding new configs